pkg/controllers/edgeconnect/phase.go

```diff
@@ -0,0 +1,54 @@
+package edgeconnect
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/status"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1alpha2/edgeconnect"
+	appsv1 "k8s.io/api/apps/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (controller *Controller) determineEdgeConnectPhase(ec *edgeconnect.EdgeConnect) status.DeploymentPhase {
+	components := []func(ec *edgeconnect.EdgeConnect) status.DeploymentPhase{
```

Comment:
It's not needed considering the current design of EC feature. No more components in the near future.

Reply:
fixed in 93c4f1d0